src/constants/lists.ts

```diff
@@ -1,20 +1,5 @@
 // the Uniswap Default token list lives here
-export const DEFAULT_TOKEN_LIST_URL = 'tokens.uniswap.eth'
+export const DEFAULT_TOKEN_LIST_URL =
```

Comment:
please change to https://www.npmjs.com/package/@fuseio/fuse-swap-default-token-list